ppr-ui/src/utils/registration-helper.ts

```diff
@@ -183,6 +185,87 @@ export async function saveDischarge (stateModel:StateModelIF): Promise<Discharge
   return apiResponse
 }
 
+/** Setup a financing statement draft for editing. Get the previously saved draft and hydrate the state model. */
+export async function setupFinancingStatementDraft (stateModel:StateModelIF, documentId:string): Promise<StateModelIF> {
+  const draft:DraftIF = await getDraft(documentId)
+  stateModel.registration.draft = draft
+  if (draft === undefined) {
+    console.error('getDraft failed: response null.')
+    return stateModel
+  }
+  if (draft !== undefined && draft.error !== undefined) {
+    console.error('getDraft failed: ' + draft.error.statusCode + ': ' + draft.error.message)
+    return stateModel
+  }
+
+  var registrationType:RegistrationTypeIF = null
+  for (const regType of RegistrationTypes) {
```

Comment:
wondering if you can do a find here?
 RegistrationTypes.find(obj => {
    return obj.registrationTypeAPI ===  draft.financingStatement.type
  })

Reply:
Great idea, I made the change.